### PR TITLE
Added a note about to big dashboards rendering, fixed chart dashboards

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
fixes https://github.com/VictoriaMetrics/helm-charts/issues/1409
fixes https://github.com/VictoriaMetrics/helm-charts/issues/1414

### CODE REVIEW

**Comment by Haleygo** [comment]
I'm not sure I understand the fix. The error message in the original issue says:
>ConfigMap "pi-monitoring-vmstack-node-exporter-full" is invalid: metadata.annotations: Too long: must have at most 262144 bytes

which indicates problem with node-exporter dashboard configmap. So if users want node-exporter dashboard and enabled it, will the error occur?

**Comment by AndrewChubatiuk** [comment]
> which indicates problem with node-exporter dashboard configmap. So if users want node-exporter dashboard and enabled it, will the error occur?

yes, argocd uses client-side apply by default and it generates anotation with content, that was last applied. Taking into account size of this dashboard, keeping this annotation will cause a problem from an issue.

**Comment by Haleygo** [comment]
But users who don't use argoCD will lose the node-exporter dashboard by default, which sounds not good.

**Review by zekker6 — APPROVED** [review]

File: charts/victoria-metrics-k8s-stack/values.yaml [hunk 1]
```diff
@@ -61,6 +61,9 @@ serviceAccount:
 dashboards:
   vmalert: false
   operator: false
+  # -- in ArgoCD using client-side apply this dashboard reaches annotations size limit and causes k8s issues without server side apply
+  # See [this issue](https://github.com/VictoriaMetrics/helm-charts/issues/1409)
```
[the remark this answers is not in the document]
Comment (AndrewChubatiuk) [reply charts/victoria-metrics-k8s-stack/values.yaml]: updated

File: charts/victoria-metrics-k8s-stack/hack/sync_dashboards.py [hunk 2]
```diff
@@ -65,6 +74,7 @@ def new_representer(dumper, data):
     "scheduler": ".Values.kubeScheduler.enabled",
     "node-rsrc-use": '(index .Values "prometheus-node-exporter" "enabled")',
     "node-cluster-rsrc-use": '(index .Values "prometheus-node-exporter" "enabled")',
+    "node-exporter-full": "false",
```
[the remark this answers is not in the document]
Comment (AndrewChubatiuk) [reply charts/victoria-metrics-k8s-stack/hack/sync_dashboards.py:77]: it allows to disable dashboard, in `dashboards.<name>` as logical OR is applied to these conditions

**Review by zekker6 — APPROVED** [review]

**Pull request merged**